### ISSUE TITLE
Flytext Beneficial status: You on you vs you on party/others

### ISSUE THREAD

**Issue by VanCluysenStijn** [Aireil/FlyTextFilter#7]
I play monk and got rid of all the stances that constantly pop up going through the rotation with beneficial status You on you turned off.
Granted this has the added effect of every status from myself on myself turned off. 
But I would still like to see the flytext in a raid from my brotherhood/mantra hitting everyone else which is not the case. 
I'm not sure if this is intended or potentially a bug? 
In short, my buffs don't show on other players and these are the only settings filtered. Bug or not?
![afbeelding](https://user-images.githubusercontent.com/38777003/225028411-873a8397-da8f-4c67-bbf9-12b5761c0e30.png)

**Comment by Aireil** [comment]
Hmm, I can't seem to reproduce this 🤔.
![j29RZTL](https://user-images.githubusercontent.com/33433913/225036875-38dac430-e408-4838-8b25-a339530a7037.png)
![aqb0xvC](https://user-images.githubusercontent.com/33433913/225036889-8f9f3ce2-9ad0-4dfb-814d-cafe529217b0.png)
The first one is on the screenshot and the second one is the filtered on me.
Could you try checking if you see them getting filtered in the log too? Also, please send what's copied when you click on `Export all types` on top of the types tab.

FWIW, to just remove the forms, you could use the `Text Blacklist` feature, but there's no target filtering on this (so it will remove the other MNKs changing forms as well):
![BqX0MnZ](https://user-images.githubusercontent.com/33433913/225038080-da6ddd70-f048-4b1b-b8d9-7d136910ce6e.png)

**Comment by VanCluysenStijn** [comment]
Your last suggestion with the blacklist is what I had implemented for Forms and Fists. 
That said, it seems I cannot reproduce the bug because it is now working as intended. 
I apologise for the inconvenience and thank you for checking in anyway!

**Comment by VanCluysenStijn** [comment]
I'm a little confused. At times it seems to be working as intended. 
But then I just ran a dungeon and sometimes "Shake it off" shows up and sometimes it doesn't. It might be range. 
(Yes, it is visible on the party frames still)
I tried using "Nascent Flash" as well with the same result.
This is the log for the Nascent on one of the party members.  
![afbeelding](https://user-images.githubusercontent.com/38777003/225088053-2ef9a5b9-aa07-4051-b52d-2ec4a05506d6.png)
And here's the export for the filters I use: 
FTF_H4sIAAAAAAAACqtWMjZVsqpWUimpLEhVslJyy6kMSa0occvMKUkt0vPNT0nN0YOKBaeWlGTmpesooKhR0lEKzi8tSk6NzC9VsjKH8QISi0oqkfj+JRmpRcVAgVodJWMzettoaERFGw3RbDQEWWBMRQtM0S1A95JhbS0AT9oNW7gBAAA=

I tried my settings with my bird and a dummy rotation and I was about to straight up send you an apology cause it was working without issue. Then I did another opener and brotherhood didn't show up. 
So It's probably something on my end that isn't properly working at times. 
I had blacklisted the Fists and Forms but it just wasn't ideal.

I'm sure your plugin is working and I apologize for the inconvenience. I think it might just be the game that sometimes shows and sometimes just does not for whatever reason. 

Thank you for taking the time to look anyway!

**Comment by VanCluysenStijn** [comment]
I turn on "log fly texts", click mantra/brotherhood, and now it works. Turn of the "log fly texts". Back to nothing. 
Is this possible?

**Comment by Aireil** [comment]
Yes, you're right, I just found a bug where the filtering wouldn't work as expected when logging is off! It would basically use `You` as a source and target, which is probably the source of your issues.

Thanks for the report, I'm going to finish something else then push an update, feel free to tell me if you still see the issue after that update.

**Comment by VanCluysenStijn** [comment]
That did the trick, thank you!
